sdk-python/awaazde/utils.py

```diff
@@ -52,29 +53,16 @@ def _get_url_params(url):
                 else:
                     break
             else:
-                print "Error in Fetching Results"
+                logging.error("Error in Fetching Results")
 
                 return calls_data
         return calls_data
 
-    def schedule_bulk_messages(self, messages):
-        """
-            Sends request to awaazde for scheduling Messages in bulk
-            :param messages: List of messages in the format as follows
-             [{
-                'templatelanguage': 3,
-                'phone_number': '+91999999999',
-                'send_on': "2018-04-04T10:00:00",
-                'values': ['dynamic11', 'dynamic21']
-                },
-                {
-                'templatelanguage': 3,
-                'phone_number': '+919999999998',
-                'values': ['dynamic12', 'dynamic22']
-                }]
-        """
+    def bulk_create_in_chunks(self, messages, limit):
```

Comment:
Explain with a code block why we have this method. This is an important util that can be used by lot of integrators